Resources/ProtobufDumper/ProtobufDumper/ProtobufDumper.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>

```

Comment:
Was this a requirement, or just habit/good practice?

Reply:
Not beyond Span